src/consensus/aft/raft.h

```diff
@@ -2193,6 +2199,15 @@ namespace aft
       if (idx <= state->commit_idx)
         return;
 
+#ifdef CCF_RAFT_TRACING
+      nlohmann::json j = {};
+      j["function"] = "commit";
+      j["new_commit_idx"] = idx;
```

Comment:
If record handling in `Traceccfraft.tla` is changed from `logline.msg..new_commit_idx` to  `logline.msg.["new_commit_idx"]`, the json key could be renamed to `commit_idx'`, i.e., syntactically reuse the prime operator for clarity.

Reply:
I lined it up on `new_configuration`: https://github.com/microsoft/CCF/blob/88499c0fcd2529acc35c37b64744c89311ba5bb3/src/consensus/aft/raft.h#L466. I think I proposed `configuration'` at the time, but you thought this might cause problems with parsing/usage in TLA+. If that's not the case, I am very happy to rename both obviously, I do agree with your point.

Reply:
Optics..., on one hand, we would loose the ability to write `logline.msg.new_commit_idx`.  On the other hand, `logline.msg["commit_idx'"] = logline.msg.commit_idx'` looks more consistent than `logline.msg.new_commit_idx = logline.msg.commit_idx'`

Reply:
Now that I see `"commit_idx'"` I am not sure, that looks easy to miss. The `new_` convention may not be the best, maybe we move stuff to `["args"]` to make it clear they're arguments to the action? That's what `commit_idx` is here, it so happens it's always applied trivially to the post state (unlike the config that gets append), but it's really an argument/parameter.

Reply:
btw I've moved this change to https://github.com/microsoft/CCF/pull/5984

Reply:
+1 for `"args"` to make it clear that this is not necessarily the next/new/primed value.

Reply:
Done in https://github.com/microsoft/CCF/pull/5984/commits/241f1b390ac88e2d5d7f98d6461631f63f050519